.github/workflows/dev-efiling-api-build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build Efiling API Image and Push to Openshift Registry
+
+on:
+  workflow_dispatch:
+    branches: [master, akroon3r-patch-2]
```

Comment:
Does your branch get removed after this?

Reply:
I need to get it into master to test it. The action isn't showing up in the actions tab for some reason.

I'm going to be editing it on the fly.